check/sh.sh

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env sh
```

Comment:
Do we need to check syntax of shell script files?

Similar to bash, what does `sh -n $file` do?

Reply:
#560